### ISSUE TITLE
508-defect-2: Edit and remove link DOM order should be consistent

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#54722]
### Point of contact

joshkimux

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

<img width="1499" alt="Screenshot 2023-03-07 at 1 10 48 PM" src="https://user-images.githubusercontent.com/14154792/223511649-fdd92012-d2e2-4af9-8971-e8e42b2165a6.png">

> On the evidence summary page, the edit link comes _before_ the title, which is inconsistent with previous edit links which come before the remove button

### Reproduction steps

1. [Go to the review evidence page](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/supporting-evidence/summary)
2. Inspect the edit link
3. Confirm that it is before the title

### Proposed solution or next steps

For consistency, the edit link should come before the remove button similar to issues:

<img width="490" alt="Screenshot 2023-03-07 at 1 12 20 PM" src="https://user-images.githubusercontent.com/14154792/223512218-1c98a58e-f44b-4f9f-85b6-c8197c4f22f4.png">

This will:
- create a consistent navigation experience for keyboard and screen reader users
- make for a better reading experience, as the user will hear the title first, description, and finally the ability to edit or remove

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by joshkimux** [comment]
@eileen-coforma and I met to talk about this ✨ we have an open question for the devs on this @Mottie or @rocio-desantiago on how much control we might have on adjusting the DOM order without messing with the visual design. Specifically:
1. Is it technically feasible (or not too complex) to keep the current visual design while having the `edit` link above the remove button and below the content in the DOM order?
2. If it isn't feasible or too complex to be worth it, @eileen-coforma and I will look into alternative visual designs ❤️

**Comment by rocio-desantiago** [comment]
@joshkimux @eileen-coforma there an example of what that looks like? I think it's doable if I'm understanding correctly. The link would be right above the button flushed to the left?

**Comment by Mottie** [comment]
I'm not sure I understand... the current DOM order is content > edit > remove. We can use CSS to visually swap the order how ever you want.

**Comment by joshkimux** [comment]
@Mottie I should have clarified— this is for the evidence summary page, where in the DOM the edit link comes before the content. As a result, it reads the edit link first, then the content, and finally the remove button.

The other page, selecting issues, has the right DOM order. We’re trying to make the two consistent with one another by fixing the evidence summary page 🙇‍♂️

@rocio-desantiago I can draft an example and post it here first thing Monday next week! 😊

**Comment by joshkimux** [comment]
### Current DOM

<img width="1259" alt="Screenshot 2023-03-13 at 7 49 04 AM" src="https://user-images.githubusercontent.com/14154792/224693766-872e12ab-31a8-4228-83f5-018647b515ea.png">

Here's a screenshot showing the DOM for the evidence summary page. Ideally, we'd want edit to come right before remove.

### Ideal DOM
<img width="1221" alt="Screenshot 2023-03-13 at 7 51 04 AM" src="https://user-images.githubusercontent.com/14154792/224694314-943ff467-4dde-4edf-baa2-822287a6f4a2.png">

### Ideal state
Ideally, we'd want to keep the visual style of the current DOM with the order of the ideal DOM. If we're unable to do that, @eileen-coforma will determine next steps from a design perspective ❤️

**Comment by Mottie** [comment]
Would it be better to move the "Edit" to be inline with the remove? That way it's consistent with the contestable issues page design
![contestable issues page showing "Test" as an entry with a March 3, 2022 decision date and below the card, an edit link and remove button are shown on a single line](https://user-images.githubusercontent.com/136959/224730181-e7124bc6-6392-4977-98ca-81df5adbded9.png)

**Comment by joshkimux** [comment]
@Mottie it would! And it would also resolve this defect ❤️ 
That being said, as it's a design change, I want to get @eileen-coforma 's blessing just in case there's something we're missing from that angle ✨

**Comment by eileen-coforma** [comment]
@rocio-desantiago @joshkimux @Mottie since it's a pattern we're using on another part of the form, you have my blessing.
